docs/user/whats-new.asciidoc

```diff
@@ -1,26 +1,267 @@
+immediate feedback on whether an alert is active (the alerting condition is met),
+inactive, or there is an error. The information about
+the errors can help you troubleshoot.
+
+[role="screenshot"]
+image::user/images/alerts-and-actions.png[Alerts and Actions UI]
+
+[float]
+[[hiding-management]]
+=== Hide management based on privileges
+
+*Stack Management* is home to an ever-growing suite of administrative
+tools to help manage your stack. These tools have historically been visible to
+all users, even if they weren’t authorized to take advantage of them. Now,
+*Stack Management* only shows the features that users are authorized to use,
+based on their assigned {ref}/security-privileges.html[cluster and index privileges].
+If a user isn’t authorized
+to see any of the *Stack Management* features, then the entire
+*Stack Management* is hidden.
+
+This highly-requested feature allows administrators to use {kibana-ref}/kibana-privileges.html#kibana-feature-privileges[feature privileges]
+instead of the deprecated Dashboard-only mode,
+which will be removed in 8.0. We also updated
+{kibana-ref}/management.html[our documentation] to make
+sure the set of required cluster and index privileges are clearly defined
+for each *Stack Management* feature.
+
+[float]
+[[copy-and-import-objects]]
+=== Copy and import of saved objects
+
+While laying the groundwork for sharing saved objects in multiple spaces,
+we overhauled the UI for copying and importing saved objects.
+When copying saved objects, you can now create objects, easily resolve conflicts,
+and get information, such as subtotals, object icons, and object titles, from the flyout.
+After importing saved objects, you’ll see a summary of what objects were created.
+
+[role="screenshot"]
+image::user/images/copy-import-objects.png[Copy to space UI]
+
+
+[float]
+[[machine-learning-enhancements]]
+=== Machine learning enhancements
+
+[float]
+==== Data frame analytics model management
+
+Enhancing the functionality for data frame analytics supervised learning,
+you can manage the trained models under **Machine Learning**. The new tab lists
+basic information on each model, with more detailed information displayed on
+row expansion, including the inference and job configuration and stats. It also
+contains a list of which ingest pipelines make use of each model.
+Additional controls allow you to search and delete models, and to view the training
+data used to create each model.
+
+[role="screenshot"]
+image::user/images/data-frame-analytics.png[Data frame analytics]
+
+[float]
+==== Per-partition categorization
+
+Categorization tokenizes a text field, clusters similar data together,
+classifies it into categories, and then detects anomalous categories in the data.
+Starting in 7.9, per-partition categorization enabled you to do categorization separately
+for every value of a partition field. With 7.10, it's now possible to configure per-partition
+categorization.
+
+[float]
+==== Feature importance
```

Comment:
Likewise if we remove the ML section, I think we need a more verbose title, something like:
```suggestion
==== More feature importance details in data frame analytics results
```